pkg/tunnelserver/tunnel.go

```diff
@@ -161,10 +165,20 @@ func (t *Authorizer) getMachine(cluster *v3.Cluster, inNode *client.Node) (*v3.N
 			}
 		}
 
+		logrus.Debugf("getMachine: looking up [%s] as node name in cluster [%s]", inNode.RequestedHostname, cluster.Name)
 		machine, err := t.machineLister.Get(cluster.Name, inNode.RequestedHostname)
 		if err == nil {
 			return machine, nil
 		}
+
+		logrus.Debugf("getMachine: looking up [%s] as RequestedHostname in cluster [%s]", inNode.RequestedHostname, cluster.Name)
+		machines, _ := t.machineLister.List(cluster.Name, labels.NewSelector())
+		for _, machine := range machines {
+			logrus.Debugf("getMachine: comparing machine.Spec.RequestedHostname [%s] to inNode.RequestedHostname [%s]", machine.Spec.RequestedHostname, inNode.RequestedHostname)
```

Comment:
we should probably remove this, this is going to be printed for all machines

Reply:
We need to be able to look into issues if Rancher won't accept the request from the node as it did before, we couldn't do that before this fix. Do you anticipate performance issues with this? Or do you want to move it after the return to only log in error situation?

Reply:
Yep my comment was only to not log it inside the loop for all the machines, we already log the start of search at L174, we could add one more statement at the end (when nothing is found/error situation as you mentioned) vs logging it for each machine